proc/beta/src/main/java/org/neo4j/gds/modularityoptimization/ModularityOptimizationStreamProc.java

```diff
@@ -71,7 +72,7 @@ protected StreamResult streamResult(long originalNodeId, long internalNodeId, No
     protected NodePropertyValues nodeProperties(
         ComputationResult<ModularityOptimization, ModularityOptimizationResult, ModularityOptimizationStreamConfig> computationResult
     ) {
-        return ModularityOptimizationProc.nodeProperties(computationResult);
+        return ModularityOptimizationProc.nodeProperties(computationResult, UUID.randomUUID().toString());
```

Comment:
Its a bit of a code smell that we need a UUID here.
Instead of passing a resultProperty to the CommunityProcCompanion, we could pass `returnChangesOnly` directly? Not a lot better though. Definitely up for other ideas here

Reply:
I have to admit that the usage of the UUID surprised me in the beginning until I saw that this approach was being used in other algorithms; I followed this approach blindly. 
I would be very grateful if you could provide me more context/info about this part :)

Reply:
Oh true, I was not aware, that in our project we already use this.
Then I would say lets make sure we get this PR over the finish line and we can internally sort out this code smell :)